samples/create-and-edit-geometries/src/main/java/com/esri/arcgismaps/sample/createandeditgeometries/screens/ButtonMenu.kt

```diff
@@ -0,0 +1,112 @@
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.arcgismaps.geometry.GeometryType
+import com.esri.arcgismaps.sample.createandeditgeometries.R
+import com.esri.arcgismaps.sample.createandeditgeometries.components.CreateAndEditGeometriesViewModel
+
+/**
+ * Composable component to display the menu buttons.
+ */
+@Composable
+fun ButtonMenu(
+    mapViewModel: CreateAndEditGeometriesViewModel
+) {
+    val rowModifier = Modifier
+        .padding(12.dp)
+        .fillMaxWidth()
+
+    val isGeometryEditorStarted by mapViewModel.geometryEditor.isStarted.collectAsStateWithLifecycle()
+
+    Row(
+        modifier = rowModifier
+    ) {
+        val vector = ImageVector
+        var expanded by remember { mutableStateOf(false) }
+        Box(
+            modifier = Modifier
+        ) {
+            IconButton(
+                enabled = !isGeometryEditorStarted,
+                onClick = { expanded = !expanded }
+            ) {
+                Icon(imageVector = Icons.Default.Create, contentDescription = "Start")
+            }
+            DropdownMenu(
+                expanded = expanded,
+                onDismissRequest = { expanded = false }
+            ) {
+                DropdownMenuItem(
+                    text = { Text("Point") },
+                    onClick = {
+                        mapViewModel.startEditor(GeometryType.Point)
+                        expanded = false
+                    }
+                )
+                DropdownMenuItem(
+                    text = { Text("Multipoint") },
+                    onClick = {
+                        mapViewModel.startEditor(GeometryType.Multipoint)
+                        expanded = false
+                    }
+                )
+                DropdownMenuItem(
+                    text = { Text("Polyline") },
+                    onClick = {
+                        mapViewModel.startEditor(GeometryType.Polyline)
+                        expanded = false
+                    }
+                )
+                DropdownMenuItem(
+                    text = { Text("Polygon") },
+                    onClick = {
+                        mapViewModel.startEditor(GeometryType.Polygon)
+                        expanded = false
+                    }
+                )
+            }
+        }
+        IconButton(
+            enabled = isGeometryEditorStarted,
+            onClick = { mapViewModel.stopEditor() }
+        ) {
+            Icon(vector.vectorResource(id = R.drawable.check_32), contentDescription = "Save Edits")
```

Comment:
Could use the default compose icons, instead of the drawable resource. 
```suggestion
            Icon(imageVector = Icons.Default.Check, contentDescription = "Save Edits")
```